Makefile

```diff
@@ -40,8 +39,16 @@ run:
 env:
 	./scripts/bootstrap-virtualenv.sh
 
-lint:
-	${PYTHONLINT} ${PYDIRS}
+lint: listoutdated flake8diff
+
+listoutdated:
+	pip list --outdated --allow-external=cafe,cloudcafe
+
+flake8diff:
+	git diff --patch --no-prefix origin/master | flake8 --diff
```

Comment:
> Yep. That's intentional. The full lint on master is just too big to fix in one PR. (run make flake8full to get an idea)

Right, I understand that we don't want to change all of master at once.  Just wondering if we want `make lint` to run only on the changes in the current branch, not on both the changes in the current branch + the changes in github master that are not currently in this branch.  So only lint the changes since where the branch diverged, not `origin/master`.

Basically instead of `git diff --patch --no-prefix origin/master`, which currently also includes integration tests changes that are on origin/master but not this branch, we can use something like `git diff --patch --no-prefix $(git merge-base master $(git rev-parse --abbrev-ref HEAD))`, or `git diff --patch --no-prefix master...$(git rev-parse --abbrev-ref HEAD)`.

I think even `git diff --patch --no-prefix origin/master...$(git rev-parse --abbrev-ref HEAD)` works.

That only lints the changes that appear in this PR.

UPDATE: note I put this in the wrong place before, deleted the other comment and moved it here instead.